Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -530,14 +551,23 @@ public void indexConcepts(String userName)
                 String searchString = fieldMap.get(search.fieldName());
                 if (searchString != null) {
                     searchString = searchString.toLowerCase();
-                    buildQuery(occur, perFieldAnalyzerWrapper, qBuild, builder, luceneFieldAnnotation, searchString);
+                    TokenStream normalizedValue = customAnalyzer().tokenStream("fieldName", searchString);
+                    CharTermAttribute cattr = normalizedValue.addAttribute(CharTermAttribute.class);
+                    normalizedValue.reset();
+                    String normalizedSearchString = null;
+                    while (normalizedValue.incrementToken()) {
+                        normalizedSearchString = cattr.toString();
+                    }
+                    normalizedValue.end();
+                    normalizedValue.close();
```

Comment:
if you set the customanalyzer into the perFieldAnalyzerWrapper then you shouldn't need this. the query will take care of it.

Reply:
Yea, this normalizer is no longer required and doesn't serve the purpose in case of exact search. Will remove this.